### ISSUE TITLE
In semver2, the metadata in the version after the "+" can be confusing

### ISSUE THREAD

**Issue by karann-msft** [NuGet/NuGetGallery#7662]
for example:
![image](https://user-images.githubusercontent.com/16904420/67807605-5b727200-fa52-11e9-8f19-97924902e521.png)

Daniel Sterling said
>I noticed on nuget.org that the versioning for the Nuget.Packaging package that I need to use in our build system has a super strange versioning scheme.  Looks like a guid is getting squished in there as well.  Is that intentional? I was just confused as to the format I needed to use in my packageReference elements in my csproj.

@scottbommarito said
> It’s unclear if the typical usage of build metadata in SemVer2 is like this—information that is not very useful for consumers—or if people are using build metadata to add meaningful content to their version string. We never analyzed this so we display it by default. we don’t allow multiple versions with the same build metadata today. This is because of SemVer2’s spec, which specifically claims that two versions with the same build metadata have identical precedence: “Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence.”

**Comment by xavierdecoster** [comment]
NuGet does NOT allow multiple versions of the same package only differing in build metadata.
If you were to push a version without build metadata, you’d encounter a conflict when a same version without build metadata exists already, and vice versa.

The build metadata is not random, and in fact provided by the publisher, so **it has meaning to the publisher** (whereas the consumer typically only cares about the normalized version - unless that consumer is the publisher), but I agree we may do something about its presentation on the site.

Perhaps we should simply show the _normalized version_ in headers on the site, and display the full version somewhere else, as part of the other metadata (e.g. above or below description).

**Comment by skofman1** [comment]
@karann-msft , @xavierdecoster , what is the action item here?

**Comment by scottbommarito** [comment]
I'd like to add that I also analyzed the existing build metadatas on NuGet.org, and it seemed like almost every single one was either a guid or some other type of build number which has zero meaning to a consumer. I think that we should not display it so prominently because it seems to mostly provoke confusion, although we may still want to show it to authors.

**Comment by xavierdecoster** [comment]
I agree with @scottbommarito: I think the build metadata is mainly useful to the publisher, less to the consumer.

The action item here would be to show it less prominently, but still make it discoverable.
I'd suggest to update the version in the title of the page to be normalized.
Perhaps we can show the full semantic version as part of the current SemVer warning?

![image](https://user-images.githubusercontent.com/880728/68194443-51ea8c00-ffb5-11e9-969a-ec09d9b4163a.png)
